Dockerfile

```diff
@@ -1,3 +1,10 @@
-FROM golang:1.21.5@sha256:135ba6c2ae2a9a022f53f3eee24ef213bc62e876811cd9582b91045168206873
+FROM golang:1.21.5@sha256:4db4aac30880b978cae5445dd4a706215249ad4f43d28bd7cdf7906e9be8dd6b
```

Comment:
This SHA256 is for the image manifest - which contains all platform images.
You can inspect it by running `docker buildx imagetools inspect golang:1.21.5-alpine`

Using this SHA256 instead of platform-specific one will mean Docker will download the pinned platform-appropriate image (e.g. linux/arm64 on Apple Silicon)

Reply:
Thanks TIL.

It would be nice to add a comment in the Dockerfile itself describing this, so that this doesn't get missed when updating the SHA.